deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -60,9 +60,9 @@ export class StackComposer {
         const mskARN = getContextForType('mskARN', 'string')
         const mskEnablePublicEndpoints = getContextForType('mskEnablePublicEndpoints', 'boolean')
         const mskBrokerNodeCount = getContextForType('mskBrokerNodeCount', 'number')
-        const sourceClusterEndpoint = getContextForType('sourceClusterEndpoint', 'string')
         const historicalCaptureEnabled = getContextForType('historicalCaptureEnabled', 'boolean')
-        const logstashConfigFilePath = getContextForType('logstashConfigFilePath', 'string')
+        const dpPipelineTemplatePath = getContextForType('dpPipelineTemplatePath', 'string')
```

Comment:
Could you update the CDK readme `deployment/cdk/opensearch-service-migration/README.md` to include these new options (and remove the old ones)